README.md

```diff
@@ -525,6 +526,7 @@ endpoints:
       tls:
         certificate-file: /path/to/user_cert.pem
         private-key-file: /path/to/user_key.pem
+        ca-file: /path/to/root-ca.pem
```

Comment:
Could you specify in the table that  ca-file is optional for non-self-signed certificate authorities, and a similar comment in the yaml example  next to `ca-file: /path/to/root-ca.pem`?

For verbosity, I want devs to keep their configuration as simple as possible, and by adding these comments, I hope it'll make it more clear that this isn't required unless it's an unrecognized/self-signed CA